mitylib/merge.py

```diff
@@ -39,12 +38,13 @@ def do_merge(debug, mity_vcf, hc_vcf, prefix=None, genome=GENOME_FILE):
         logger.error("The VCF files use mitochondrial contigs")
         sys.exit()
 
-    prefix = create_prefix(hc_vcf, prefix)
+    prefix = "placeholder"
```

Comment:
Is this meant to be like this or was this placeholder for you to change it later?